manifests/reverse_proxy.pp

```diff
@@ -67,4 +67,10 @@
     ],
     *                      => $vhost_params,
   }
+
+  if $foreman_proxy_content::enable_pulp3 {
```

Comment:
I would move this into `init.pp` for now and also add `and pulp_master` to the conditional to only do this on a server.